pkgs/development/tools/misc/blisp/default.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  IOKit,
+  testers,
+  blisp,
+}:
```

nixpkgs manual sadly states that this is the formatting we are supposed to use, but our formatting rfc (rfc-101) preferes the one you are using, so for now we have to do that:
```suggestion
{ lib
, stdenv
, fetchFromGitHub
, cmake
, IOKit
, testers
, blisp
}:
```